Refactor and Fix error and add some feature

UranusLin opened the pull request:
feat: Refactor architecture
feat: Add requirement.txt
feat: two mode for browse
fix: Mac open Tor not found
fix: change IP connection refused

Commit:
feat: project architecture
feat: Add requirement.txt
feat: two mode for browse
fix: Mac open Tor not found
fix: change IP connection refused

UranusLin committed:
Create LICENSE.md

UranusLin committed:
Merge pull request #1 from UranusLin/add-license-1

Create LICENSE.md

Commit:
feat: add new mode with 3 it's allow to close tor and reopen tor for some website

UranusLin committed:
Merge pull request #2 from UranusLin/add_close_open

feat: add new mode with 3 it's allow to close tor and reopen tor for …

jakedent commented on LICENSE.md:
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2022 MorrisLin
```
Please add my name (Jacob Dent). Thanks.

UranusLin replied:
already add thanks.

jakedent commented on service.py:
```diff
@@ -0,0 +1,53 @@
+
+
+# browse for infinity
+def browse_by_close_open(url: str):
```
Please could the browse_by_infinity method be extended to include an optional argument and some logic to check if the argument has been passed, to then call start_browse() if required in the loop? Lines 25 to 31 are very similar to lines 6-12 and the only difference being start_browser() is called instead of change_ip(). Thanks.

UranusLin replied:
I already refactor this part to give cmd to control different way.

jakedent commented on utils.py:
```diff
@@ -0,0 +1,65 @@
+    try:
+        os.system('osascript -e \'quit app "Tor Browser.app"\'')
+    except Exception as s:
+        print("Unable to start Tor Browser: {0}".format(s))
```
Please change start to quit. Thanks.

jakedent reviewed (changes requested):
Cool update man. Thanks!!

If you could please make the changes suggested will be happy to merge and add another successful pull request to your list! If you manage to remove the browse_by_close_open method by way of extending the browse_by_infinity method, please update line 14, add nested logic to line 15 and remove lines 18 and 19. 

Thanks again.

Commit:
feat: LICENSE add Jacob Dent
feat: refactor for browse_by_infinity and remove browse_by_close_open
feat: refactor log wording

UranusLin committed:
Merge pull request #3 from UranusLin/add_close_open

refactor from suggest
